### PR TITLE
[add] plus and currency-use icons

### PR DESCRIPTION
**Pull request by finalprototype**
screenshot 2017-05-24 12 51 30

### CODE REVIEW

**Comment by sashako** [comment]
beautiful!

File: docs/styles/_icons-section.scss [hunk 1]
```diff
@@ -160,4 +160,12 @@
   .icon-email {
     @include icon('email');
   }
+
+  .icon-plus {
+    @include icon('plus');
+  }
+
+  .icon-currency-usd {
+    @include icon('currency-usd');
```
Comment (nhinsch) [inline docs/styles/_icons-section.scss]: Super nit: Is `currency-usd` the most obvious name for this icon? I might just call it `dollar` or `dollar-sign`. Looks good though, :ship:

Comment (finalprototype) [inline docs/styles/_icons-section.scss]: In terms of easily read and called global currencies, no. Makes more sense to specify the icon type-subtype. ie: `currency-eur`, `currency-gbp`, etc

Comment (nhinsch) [inline docs/styles/_icons-section.scss]: :+1:

**Pull request merged**